ksqldb-rest-app/src/main/java/io/confluent/ksql/api/server/QueryStreamHandler.java

```diff
@@ -164,7 +164,7 @@ private CommonRequest getRequest(final RoutingContext routingContext) {
       if (!ksqlRequest.isPresent()) {
         return null;
       }
-      sql = ksqlRequest.get().getKsql();
+      sql = ksqlRequest.get().getUnmaskedKsql();
```

Comment:
I think this is going to NPE -- it's used from the `/query-stream` endpoint which doesn't go through OldApiUtils. (The class is also used from `/query` which does go through OldApiUtils, so it won't necessarily always NPE.)

Might be good to be defensive on these other resources, similar to what's done for KsqlResource right now (set masked if needed).